app/views/Layout.scala.html

```diff
@@ -31,33 +31,35 @@
         hmrcTimeoutDialog: HmrcTimeoutDialog,
         h2: components.h2,
         p: components.p
-
 )
 
-@(pageTitle: Option[String] = None,
+@(
+        pageTitle: Option[String] = None,
```

Comment:
is this over indented?

Reply:
Done. It was aligned with parameters of @this(. But now, both have be done and reduced one extra tab